app/controllers/purchases_controller.rb

```diff
@@ -0,0 +1,38 @@
+class PurchasesController < ApplicationController
+  before_action :prevent_duouble_purchase, only: [:create]
+  before_action :release_date_came?, only: [:create]
+
+  def new
+  end
+
+  def create
+    @webbook = Webbook.find(params[:id])
+
+    @stripe_form = StripeForm.new(params[:stripeEmail], params[:stripeToken], @webbook)
+    @stripe_form.save!
+
+    @purchase_history_webbooks = PurchaseHistoryWebbook.new
+    @purchase_history_webbooks.add_purchase_history(current_user, params[:purchase_history_id], @webbook.id, @stripe_form.charge[:id])
+
+    PurchaseMailer.creation_email(current_user).deliver_now
```

Comment:
まだメール作り込んでないけど、多分買ったものの情報は必要かと。

```suggestion
    PurchaseMailer.creation_email(@purchase_history_webbook).deliver_now
```